src/Microsoft.AspNet.Mvc.Core/ViewContext.cs

```diff
@@ -97,12 +124,30 @@ public dynamic ViewBag
             }
         }
 
+        /// <summary>
+        /// Gets or sets the view.
+        /// </summary>
         public IView View { get; set; }
 
+        /// <summary>
+        /// Gets or sets the <see cref="ViewDataDictionary"/>.
+        /// </summary>
         public ViewDataDictionary ViewData { get; set; }
 
+        /// <summary>
+        /// Gets or sets the <see cref="TextWriter"/> used to write the output.
+        /// </summary>
         public TextWriter Writer { get; set; }
 
+        /// <summary>
+        /// Gets or sets the path of the View file currently being executed.
+        /// </summary>
+        /// <remarks>
+        /// The rendering of a view may involve one or more files being rendered (e.g. Layouts, Partials etc).
+        /// This property contains the path of the file currently being rendered.
```

Comment:
nit: remove first "being rendered" and (perhaps) add "specific" before "file currently"